web/e2e/config/configs.js

```diff
@@ -0,0 +1,124 @@
+// @flow
+
+export type SeleniumCapabilitiesType = {|
```

Comment:
maybe it would make sense to include a type string, e.g. `type: 'selenium'`

Reply:
All of this are selenium configs. So we rather name this `DefaultCapabilitesTypes`?

Reply:
> All of this are selenium caps. So we rather name this DefaultCapabilitesTypes?

I prefer having the name explicitly here as otherwise this information is lost, as the capabilities are only used for setup and shouldn't be touched later in the tests.

> maybe it would make sense to include a type string, e.g. type: 'selenium'

What would be the advantage of that? :)

Reply:
I thought it would make it clearer which kind of caps it is. But nvm, everything good as it is.